pkg/tfshim/sdk-v2/proposed_new_test.go

```diff
@@ -72,3 +73,43 @@ func TestProposedNew(t *testing.T) {
 		})
 	}
 }
+
+func TestCtyTurnaround(t *testing.T) {
+	type testCase struct {
+		value cty.Value
+	}
+
+	testCases := []testCase{
```

Comment:
Can we add some tests for nested types: `cty.ListValEmpty(cty.ListValEmpty(cry.NumberIntVal(42)))`?

Reply:
Sure.